Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```diff
@@ -845,10 +852,18 @@
     fireModes:
     - proto: BulletLaserSpread
       fireCost: 150
+      color: "#5fcde4"
     - proto: BulletLaserSpreadNarrow
       fireCost: 200
+      color: "#5fcde4"
     - proto: BulletDisablerSmgSpread
       fireCost: 120
+      color: "#ff0006"
+    clothingVisuals:
+      back:
+      - state: equipped-BACKPACK-color
+      suitstorage:
+      - state: equipped-SUITSTORAGE-color
```

Comment:
The equipped sprites are using a greyer sprite which causes the color to be very subdued, losing contrast and readability. Should be made brighter. Feel free to compare to the base sprite's coloration. 
<img width="708" height="727" alt="image" src="https://github.com/user-attachments/assets/a8e53860-971f-4316-b8e9-f0d80d84de01" />